assets/elm/DrinksCarousel.elm

```diff
@@ -129,12 +129,12 @@ subscriptions model =
 view : Model -> Html Msg
 view model =
     div [ class "relative" ]
-        [ img [ src "images/up-chevron.svg", alt "left arrow", class "dn db-ns f1 b pointer absolute-vertical-center left-2 rotate-270 h1", onClick CarouselLeft ] []
+        [ img [ src "images/up-chevron.svg", alt "left arrow", class "z-max f1 b pointer absolute-vertical-center left-1 left-2-ns rotate-270 h1", onClick CarouselLeft ] []
```

Comment:
```suggestion
        [ img [ src "images/up-chevron.svg", alt "left arrow", class "z-1 f1 b pointer absolute-vertical-center left-1 left-2-ns rotate-270 h1", onClick CarouselLeft ] []
```

Reply:
This still appears as it should with `z-1` and it's better practice to only sent a z-index as high as you need to, so I'd favour z-1 over z-max where possible.